dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/impl/AsyncResponseImpl.java

```diff
@@ -348,7 +348,11 @@ public synchronized void handleTimeout() {
     private void initContinuation() {
         ContinuationProvider provider =
             (ContinuationProvider)inMessage.get(ContinuationProvider.class.getName());
-        cont = provider.getContinuation();
+        if (provider != null) {
+            cont = provider.getContinuation();
+        } else {
+            throw new IllegalArgumentException("All servlets and servlet filters must support async!");
```

Comment:
same as above.  Also, not a huge deal, but you can save a line of code by doing:
```
if (provider == null) {
    throw new IllegalArgumentException("Continuation not supported. Please ensure that all servlets and servlet filters must support async operations");
}
cont = provider.getContinuation();